services/core-web/src/components/mine/Summary/MineWorkInformation.tsx

```diff
@@ -0,0 +1,267 @@
+                        </Typography.Text>
+                        <br />
+                        <Typography.Text>
+                          6.2.2 The manager shall give notice to an inspector of intention to stop
+                          [any mining activity] in, at, or about a mine, permanently, indefinitely,
+                          or for a definite period exceeding 30 days, and except in an emergency,
+                          the notice shall be not less than seven days.
+                        </Typography.Text>
+                      </>
+                    }
+                  />
+                </>
+              }
+            >
+              {formatDate(info.work_stop_date) || Strings.NOT_APPLICABLE}
+            </Descriptions.Item>
+          </Descriptions>
+          <Descriptions column={1} colon={false}>
+            <Descriptions.Item label="Comments" span={1}>
+              {info.work_comments || Strings.NOT_APPLICABLE}
+            </Descriptions.Item>
+          </Descriptions>
+          {/* NOTE: Ant Design has no easy way to right-align "Descriptions" so plain HTML tags are used here. */}
+          <span style={{ float: "right", display: "inline-flex" }}>
+            <div className="inline-flex padding-sm" style={{ marginRight: 10 }}>
+              <p className="field-title">Updated By</p>
+              <p>{info.updated_by}</p>
+            </div>
+            <div className="inline-flex padding-sm">
+              <p className="field-title">Last Updated</p>
+              <p>{formatDateTime(info.updated_timestamp)}</p>
+            </div>
+          </span>
+        </Col>
+        <Col span={2}>
+          <span style={{ float: "right" }}>
+            <AuthorizationWrapper permission={Permission.EDIT_MINES}>
+              <Button
+                type="primary"
+                size="small"
+                ghost
+                onClick={() => openAddEditMineWorkInformationModal(info)}
+              >
+                <EditOutlined className="icon-lg icon-svg-filter" />
+              </Button>
+            </AuthorizationWrapper>
+            <AuthorizationWrapper permission={Permission.EDIT_MINES}>
+              <Popconfirm
+                placement="topLeft"
+                title="Are you sure you want to delete this record?"
+                onConfirm={() => deleteMineWorkInformation(info.mine_work_information_guid)}
+                okText="Delete"
+                cancelText="Cancel"
+              >
+                <Button type="primary" size="small" ghost>
+                  <DeleteOutlined className="icon-lg icon-svg-filter" />
+                </Button>
+              </Popconfirm>
+            </AuthorizationWrapper>
+          </span>
+        </Col>
+      </Row>
+    </List.Item>
+  );
+
+  const dataSource = showAll
+    ? props.mineWorkInformations
+    : !isEmpty(props.mineWorkInformations)
+    ? [props.mineWorkInformations[0]]
+    : [];
```

Comment:
now I might be getting picky but did the linter run? I was wondering how this syntax could be real until I noticed that the indentation is wacky. 😂

Reply:
The linter definitely ran here. It looks like on my laptop the indentation should look like this

```
  const dataSource = showAll
    ? props.mineWorkInformations
    : !isEmpty(props.mineWorkInformations)
      ? [props.mineWorkInformations[0]]
      : [];
```
but the linter changes it to what we see in the pr.

Reply:
_Gross_. But I accept this explanation. 👍